apps/ledger-live-desktop/src/renderer/components/PostOnboardingHub/PostOnboardingHubContent.tsx

```diff
@@ -39,7 +38,11 @@ const PostOnboardingHubContent = () => {
         <Link onClick={handleSkipButton} data-test-id={"postonboarding-hub-skip-button"} mt={5}>
           {t("postOnboarding.postOnboardingContent.skipLink")}
         </Link>
-      ) : null}
+      ) : (
+        <ButtonV3 big variant="main" mt={3} onClick={handleOnboardingOver} event="button_clicked">
```

Comment:
`event="button_clicked"` this will trigger twice Analytics if i'm not wrong

Reply:
Oh yeah indeed, you are right, nice one 🙏